packages/@okta/courage-dist/types/courage/framework/Collection.d.ts

```diff
@@ -0,0 +1,43 @@
+import Backbone, { _Result } from 'backbone';
```

Comment:
do we need to check in all those types ?

Reply:
we do need to check in the courage types, since only we will have the original source and be able to build it.